en/components/grid.md

```diff
@@ -377,40 +535,178 @@ Defining handlers for these event emitters is done using declarative event bindi
-| `focusRow`                                       | Focuses the grid row at `index`.                         |
-| `filterData`                                     | Filter data by search term and column                    |
-| `addRow`                                         | Add record to the grid data container                    |
-| `deleteRow`                                      | Remove record from the grid data container               |
-| `updateRow`                                      | Update record from the grid data container               |
-| `updateCell`                                     | Update grid cell by index, column field and passed value |
-| `sortColumn`                                     | Sort grid column                                         |
-| `paginate`                                       | Change the current page by passed number                 |
+|Signature|Description|
+|--- |--- |
+|`getColumnByName(name: string)`|Returns the column object with field property equal to `name` or `undefined` if no such column exists.|
+|`getCellByColumn(rowIndex: number, columnField: string)`|Returns the cell object in column with `columnField` and row with `rowIndex` or `undefined`.|
+|`addRow(data: any)`|Creates a new row object and adds the `data` record to the end of the data source.|
+|`deleteRow(rowIndex: number)`|Removes the row object and the data recrod corresponding to it from the data source.|
+|`updateRow(value: any, rowIndex: number)`|Updates the row object and the data source record with the passed value.|
+|`updateCell(value: any, rowIndex: number, column: string)`|Updates the cell object and the record field in the data source.|
+|`filter(column: string, value: any, condition?, ignoreCase?: boolean)`|Filters a single column. Check the available [filtering conditions](#filtering-conditions)|
+|`filter(expressions: Array)`|Filters the grid columns based on the provided array of filtering expressions.|
+|`filterGlobal(value: any, condition? ignoreCase?)`|Filters all the columns in the grid.|
+|`clearFilter(name?: string)`|If `name` is provided, clears the filtering state of the corresponding column, otherwise clears the filtering state of all columns.|
+|`sort(name: string, direction, ignorecase)`|Sorts a single column.|
+|`sort(expressions: Array)`|Sorts the grid columns based on the provided array of sorting expressions.|
+|`clearSort(name?: string)`|If `name` is provided, clears the sorting state of the corresponding column, otherwise clears the sorting state of all columns.|
+|`previousPage()`|Goes to the previous page if paging is enabled and the current page is not the first.|
+|`nextPage()`|Goes to the next page if paging is enabled and current page is not the last.|
+|`paginate(page: number)`|Goes to the specified page if paging is enabled. Page indexes are 0 based.|
+|`markForCheck()`|Manually triggers a changed detection cycle of the grid and its children.|
```

Comment:
change detection?  Or does it trigger a detection cycle that has changed already?